rmgpy/rmgobject.pyx

```diff
@@ -115,3 +115,21 @@ cdef class RMGObject(object):
                 except ValueError:
                     pass
         self.__init__(**data)
+
+cpdef expand_to_dictionaries(obj):
+    if isinstance(obj, list):
+        return [expand_to_dictionaries(x) for x in obj]
+
+    elif isinstance(obj, dict):
+        for key, value in obj.iteritems():
+            obj[key] = expand_to_dictionaries(value)
```

Comment:
what if the key is an object as well?

Reply:
Are we sure this won't change the original object?

Reply:
Just to document this here, we figured out that dictionaries themselves are not hashable, and thus cannot serve as keys for other dictionaries. Therefore, we can not simply expand objects to dictionaries if they are being used as keys for a dictionary. 

Max and I had some ideas for how we might tackle this, but this is a subject for a later PR due to its complexity and the fact that no RMGObjects currently have attributes that are dictionaries with objects as keys (that is until we make Molecule an RMGObject)